columnflow/tasks/framework/parameters.py

```diff
@@ -31,16 +32,28 @@ class SettingsParameter(law.CSVParameter):
     def parse_setting(cls, setting: str) -> tuple[str, float | bool | str]:
         pair = setting.split("=", 1)
         key, value = pair if len(pair) == 2 else (pair[0], "True")
+        if ";" in value:
```

Comment:
Would it make sense to make the delimiter (in this case `;`) a member variable of the class?

Reply:
Yes! In that way we do not need to update the tests when changing this! I'll change this tomorrow

Reply:
Nice, thanks!